charts/argo-cd/templates/_helpers.tpl

```diff
@@ -40,6 +40,17 @@ Create dex name and version as used by the chart label.
 {{- printf "%s-%s" (include "argo-cd.fullname" .) .Values.dex.name | trunc 63 | trimSuffix "-" -}}
 {{- end -}}
 
+{{/*
+Create Dex server endpoint
+*/}}
+{{- define "argo-cd.dex.server" -}}
```

Comment:
Support for new HTTP / HTTPS for dex server. Tested via OIDC provider config on RC3